inc/command/class-command.php

```diff
@@ -213,11 +214,18 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 	 */
 	protected function codecept( InputInterface $input, OutputInterface $output ) {
 		$options = $input->getArgument( 'options' );
-		$tests_folder = $input->getOption( 'path' );
+		// Codeception command runs within `vendor` directory, so paths are relative to that.
+		$tests_folder = '../' . $input->getOption( 'path' );
 		$output_folder = $input->getOption( 'output' );
-		$run_headless_browser = $input->getOption( 'browser' );
+		$run_headless_browser = $input->getOption( 'browser' ) ?: 'chrome';
 		$use_chassis = $input->getOption( 'chassis' );
 		$project_subdomain = $this->get_project_subdomain();
+		$subsubcommand = $input->getArguments()['options'][0] ?? '';
```

Comment:
For the most part this'll be fine but people could throw all sorts at it like `composer dev-tools codecept -p some/path bootstrap`. We can't always control the order options come in so if we're only looking for subsubcommand to match bootstrap this can probably just be an `in_array()` check.

Reply:
`InputInterface` does a good job clearing parameter arguments, so even in that case, `bootstrap` will still be returned as the first array element. An `in_array` call can match directories as well.

However, `hasParameterOption` seems to be cleaner, so will go with that.

Reply:
Ahh good to know, was thinking of it like the raw `$argv` data.